src/pages/onboarding/[step].tsx

```diff
@@ -0,0 +1,60 @@
+/** @jsx jsx */
+import { jsx, Grid } from "theme-ui"
+import { TabNavigation } from "../../components/tab-navigation"
+import { navigate } from "gatsby"
+import { ChooseEditor } from "../../components/onboarding/choose-editor"
+import { ChooseSites } from "../../components/onboarding/choose-sites"
+
+export interface IProps {
+  params: {
+    step: OnboardingStep
+  }
+}
+
+type OnboardingStep = "editor" | "sites"
+
+const ONBOARDING_STEPS_ORDER: OnboardingStep[] = [`editor`, `sites`]
+
+export default function OnboardingWizard({
+  params: { step },
+}: IProps): JSX.Element {
+  return (
+    <Grid
+      css={{
+        height: `100vh`,
+        width: `100vw`,
+        gridTemplateRows: `32px 1fr`,
+      }}
+    >
+      <TabNavigation />
+      <main
+        sx={{
+          overflowY: `auto`,
+          px: `15%`,
+          pb: 10,
+          pt: [12, null, null, null, `10rem`],
+          fontFamily: `sans`,
+        }}
+      >
+        {step === `editor` && (
```

Comment:
I'm not convinced of the value of using unified routes in this situation. There are a fixed set of steps, each with its own component. I'd rather create a wrapper component to share, and a page template for each step